internal/repositories/storage.go

```diff
@@ -125,24 +141,24 @@ func (f InMemoryStorage) GetFullURL(id string) (string, error) {
 	return f.dataURL[id][0], nil
 }
 
-func (f FileStorage) AddShort(body string, s string, idUser string) error {
+func (f FileStorage) AddShort(item domain.URLForAddStorage) (string, error) {
```

Comment:
По логике мы просто сохраняем данные в базу и ничего не меняем в передаваемой структуре, мы и до вызова этого метода знали, что короткий урл можно достать из item.ShortURL, с моей точки зрения возвращаемый string излишен
```suggestion
func (f FileStorage) AddShort(item domain.URLForAddStorage) error {
```